### PR TITLE
Package is now on NPM 🎉

### PR DESCRIPTION
**Pull request by alexbchr**
Renamed `@hansgx` references to `@trampoline`.

Deployed package on [NPM](https://www.npmjs.com/package/@trampoline/cubes).

@hansgx What about referencing the Storybook as the Website in `package.json`?

We could refer to the  [Storybook](https://5eebb872a669600022881133-ttvmbedqny.chromatic.com/?path=/story/actions-badge--default) generated by Chromatic.

Reviewed the README a bit, but there is more documentation work to be done in it. (Mainly removed DS references and removed no longer valid stuff).

### CODE REVIEW

**Comment by hansgx** [comment]
@alexbchr Let's keep the repo as the website. We can link to Storybook from the README

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
```
Comment (hansgx) [inline LICENSE:1]: Should this be a `.md`?

**Comment by alexbchr** [comment]
> @alexbchr Let's keep the repo as the website. We can link to Storybook from the README

I meant we could put the Storybook as [Homepage](https://docs.npmjs.com/files/package.json#homepage). Which appears as a separate link from the repository.

File: LICENSE (hunk 1 above)
Comment (alexbchr) [reply LICENSE:1]: According to GitHub, yes it should be! Renamed :rocket:

**Pull request merged**